sherpa/astro/io/crates_backend.py

```diff
@@ -952,8 +952,7 @@ def get_pha_data(arg, make_copy=True, use_background=False):
         # Chandra Level 3 PHA files
         for ii in range(phadataset.get_ncrates()):
             block = phadataset.get_crate(ii + 1)
-            hduclas2 = block.get_key('HDUCLAS2')
-            if hduclas2 is not None and hduclas2.value == 'BKG':
+            if (_try_hdr_key(block, 'HDUCLAS2') == 'BKG'):
```

Comment:
Yes, we are happy.

Since this is the release branch for CIAO, and we are still not clear, technically we shouldn't merge this PR. I was thinking I would merge this branch in the 4.8.2 release branch, though, so we get these changes. What I expect is that if we merge 4.8.2 first (i.e. we release 4.8.2 before b1.5 is "released"), then CIAO4.9b1.5 will result merged as well, which is the same as if we merged it now. However, what's more likely to happen is that the b1.5 CIAO cut will be "released" before that. Then it's just a question of whether we want to rebase the 4.8.2 release branch or not. The double quotes in "released" (for CIAO) make me think we could do ad-hoc stuff in this specific case, but it's a good exercise nevertheless, and I am not a big fan of ad-hoc anyway :)